packages/nextcloud/lib/src/webdav/path_uri.dart

```diff
@@ -12,6 +12,12 @@ class PathUri {
     required this.pathSegments,
   });
 
+  /// Base path used for dav on the server
+  factory PathUri.dav() => PathUri.parse('/remote.php/dav');
+
+  /// Base path used for webdav on the server
+  factory PathUri.webdav() => PathUri.parse('/remote.php/webdav');
+
```

Comment:
The `PathUri` should be a `Uri` like class and not provide any paths itself.

Maybe create an enum with a getter that returns the uri.

Reply:
IMO this can just be hardcoded in the constructors of the WebDavClient.